dev-tools/packaging/packages.yml

```diff
@@ -398,3 +403,14 @@ specs:
         files:
           '{{.BeatName}}{{.BinaryExt}}':
             source: ./{{.XPackDir}}/{{.BeatName}}/build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
+
+    - os: linux
+      types: [docker]
```

Comment:
why do we a second docker setup here? Can't we just add the `docker_ubi_spec` to the already existing one at line 397?

Reply:
This will create a second image, but this time with ubi7 as the base image. In the end we want to keep the existing centos based images and add a UBI based image.

Reply:
For licensed beats, we will have a new image with `-ubi` in the name that's based on `ubi7`.  We'll also have the current container based on `centos:7`.

```
$ docker image ls -a | grep filebeat                                                                                        add-ubi
docker.elastic.co/beats/filebeat                8.0.0                  736ac93d4a39        About a minute ago   352MB
docker.elastic.co/beats/filebeat-ubi            8.0.0                  9efa531bcada        About a minute ago   354M
```